Source/DafnyRuntime/DafnyRuntime.cs

```diff
@@ -5,6 +5,11 @@
 //
 //-----------------------------------------------------------------------------
 
+// When --include-runtime is true, this file is directly prepended
```

Comment:
I'm a bit confused. Isn't this file generated? If so, how does this comment persist?